app/services/market_analysis.py

```
@@ -106,18 +106,100 @@ class ExchangeConfigurations:
+            "ticker": "/products/{symbol}/ticker",
+            "products": "/products",
+            "stats": "/products/{symbol}/stats",
+            "orderbook": "/products/{symbol}/book"
+        },
+        "rate_limit": 600,
+        "weight_limits": {
+            "ticker": 1,
+            "stats": 1,
+            "orderbook": 1
+        }
+    }
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)